activesupport/lib/active_support/test_case.rb

```diff
@@ -45,6 +45,41 @@ def test_order
 
         test_order
       end
+
+      def run(reporter, options = {})
+        if filter = line_filter_for(options[:patterns] || [])
```

Comment:
why do we need this method here, and not in the Runner?
If we had this method in the runner, we could called it from the rails_minitest_plugin.
And tests that only extend Minitest::Test would also work.